k8s/upgrades/0.8.0-0.8.1/cstor_pool_update.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+##################################################################
+# STEP: Get SPC name as argument                                 #
+#                                                                #
+# NOTES: Obtain the pool deployments to perform upgrade operation#
+##################################################################
+
+function usage() {
+		echo
+		echo "Usage:"
+		echo
+		echo "$0 <spc-name> <openebs-namespace>"
+		echo
+		echo "  <spc-name> Get the SPC name using: kubectl get spc"
+		echo "  <openebs-namespace> Get the namespace where openebs"
```

Comment:
can we rephrase as below:
`<openebs-namespace> Get the namespace where pool pods corresponding to SPC are deployed`

Reply:
Can namespace be derived in the code itself. Otherwise, we are asking user to learn the internals before proceeding with upgrade.

Reply:
I tried to find one few months back, but, I don't think there is a way to get this correctly

Reply:
@AmitKumarDas I am not able to get namespace from the process. I think better to get the namespace from the user.